CORE
main.py

array bounds violated: array `arr' upper bound